mautrix_facebook/analytics.py

```diff
@@ -1,41 +1,44 @@
 from __future__ import annotations
 
+from urllib.parse import urlunparse
 import logging
 
-from yarl import URL
```

Comment:
The imports and type hint of `analytics_url` also need to be changed (github just isn't nice for suggestions across many places)